src/lingrex/regularity.py

```diff
@@ -51,7 +51,7 @@ def regularity(wordlist, threshold=3, ref="cogid", min_refs=3, word_threshold=0.
             wordlist.msa[ref].items()):
         scores = []
         for idx in range(len(msa["alignment"][0])):
-            if (cogid, idx) not in wordlist.patterns:
+            if (cogid, idx) not in wordlist.patterns:  # pragma: no cover
```

Comment:
What does this mean?

Reply:
The `# pragma: no cover`? This means that the line is not taken into account for the coverage report. You'd typically do that e.g. for lines raising an exception in programs to be run by end users. In that case a test wouldn't add much.